src/core/Akka.TestKit/TestKitBase_AwaitConditions.cs

```diff
@@ -293,10 +293,59 @@ protected static async Task<bool> InternalAwaitConditionAsync(Func<Task<bool>> c
             return true;
         }
 
-        private static void ConditionalLog(ILoggingAdapter logger, string format, params object[] args)
+        protected void AwaitCond(Func<bool> p, TimeSpan? max = null, TimeSpan? interval = null, string message = "")
```

Comment:
Don't we already have an `AwaitCondition` method or does this do something different?

Reply:
The behavior of `AwaitCondition` is different from the JVM, that's why I tried a straight port instead. No luck, keeps failing.

Reply:
`AwaitCondition` arbitrarily calculates the `interval`(when not specified) [as a 10th](https://github.com/akkadotnet/akka.net/blob/dc232c6c38dd346631a5d15605be40fca8633244/src/core/Akka.TestKit/TestKitBase_AwaitConditions.cs#L45) of `max`. It should be a fixed `100 ms`, otherwise tests ported from the JVM might not behave as expected.